specification/compute/resource-manager/Microsoft.ContainerInstance/2017-08-01-preview/examples/ContainerGroupsListAll.json

```diff
@@ -0,0 +1,60 @@
+{
+  "parameters": {
+    "subscriptionId": "subid",
+    "api-version": "2017-08-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/subid/resourceGroups/demo/providers/Microsoft.ContainerInstance/containerGroups/demo1",
+            "location": "west us",
+            "name": "demo1",
+            "properties": {
+              "containers": [
+                {
+                  "name": "demo1",
+                  "properties": {
+                    "command": null,
+                    "environmentVariables": [
+                      {
+                        "name": "a",
+                        "value": "ab"
+                      }
+                    ],
+                    "image": "nginx",
+                    "instanceView": null,
+                    "ports": [
+                      {
+                        "port": 80
+                      }
+                    ],
+                    "resources": null
+                  }
+                }
+              ],
+              "imageRegistryCredentials": null,
+              "ipAddress": {
+                "ip": "10.0.0.0",
+                "ports": [
+                  {
+                    "port": 80,
+                    "protocol": "TCP"
+                  }
+                ]
+              },
+              "osType": "Linux",
+              "provisioningState": "Succeeded",
+              "restartPolicy": null,
+              "state": null
+            },
+            "resourceGroup": "demo",
```

Comment:
this property is not documented in the Swagger (so it won't be accessible in statically typed SDKs), just making sure you didn't accidentally miss it